documentation/homework/week1/todo-app/src/components/todo-item.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+export default class todoItem extends Component {
+  render() {
+    const { description, deadLineDate, isDone} = {
```

Comment:
I actually have a very hard time understanding what is going on here. If you want to split out variables from `this.props`, usually we do it this way:

    const { description, deadLineDate, isDone } = this.props

If you want to do further modification, like using `toDateString()` etc, you should do that afterwards so it's more clear what's happening.